src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -865,7 +865,8 @@ bool clang_c_convertert::get_type(const clang::Type &the_type, typet &new_type)
     if(get_type(arr.getElementType(), sub_type))
       return true;
 
-    new_type = array_typet(sub_type, gen_one(index_type()));
+    new_type = array_typet(sub_type, gen_zero(index_type()));
+    new_type.set("incomplete", "true");
```

Comment:
why an incomplete array has size one?

Reply:
It is zero. The previous implementation used to go with one.